catppuccin-steam.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash 
+set -euo pipefail
+IFS=$'\n\t'
+
+# Install dependencies
```

Comment:
We should really ask the user for confirmation before installing packages

Reply:
Also, just noticed, but arch should have "--needed" appended to the install command-- that way, it doesn't reinstall stuff it doesn't need